examples/standalone/benchmarks/process_profiling.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+SCRIPT=`realpath $0`
+SCRIPT_DIR=`dirname $SCRIPT`
+ROOT_DIR="$(dirname "$(dirname "$(dirname "$SCRIPT_DIR")")")"
+
+# activate Python environment
+venv="$ROOT_DIR/venv"
```

Comment:
could pass this as an argument? though probably fine to leave as-is

Reply:
Since this script is intended to be able to run manually, I tried using the same logic as in `run_on_daint.sh` and avoided baking in too many arguments.

Reply:
Sure, it just means we need to assume the virutalenv to be used is always rootdir/venv, while there could be use cases to use others. but no need to change, someone can if they actually need that use case

Reply:
Changed.